eoscostarica/eosio-spend-explorer
# feat(webapp): move delegate names below graph

### Move delegate names below graph

### What does this PR do?

- Resolve #114 
- Resolve #102
- Resolve #103
- Resolve #120 
- Resolve #117 
- Resolve #115

### Steps to test

1. Run the project
2. Open in your browser localhost:3000
3. Click on income in the sidebar
4. See the new changes

- Now
![image](https://user-images.githubusercontent.com/56526123/190278880-d01cdfaf-06e8-4db4-843a-3c37cb4e0413.png)

- Before
![image](https://user-images.githubusercontent.com/56526123/190278939-1a434f55-2e61-4b2b-a1d3-bbb472da85d9.png)


#### CheckList
